lib/controllers.js

```diff
@@ -98,3 +99,13 @@ Controllers.deleteStrategy = async (req, res) => {
 	const strategies = await main.listStrategies();
 	helpers.formatApiResponse(200, res, { strategies });
 };
+
+Controllers.userByOAuthId = async (req, res) => {
+	const userId = await main.getUidByOAuthid(req.params.provider, req.params.oAuthId);
+	if (!userId) {
+		helpers.formatApiResponse(404, res);
+	}
```

Comment:
missing a return here

Reply:
Alternatively, you can `return next()` in order to invoke the regular api 404 handler. You'll have to add `next` to the method parameters.